nzedb/NZBImport.php

```diff
@@ -297,15 +297,25 @@ protected function scanNZBFile(&$nzbXML, $useNzbName = false)
 			// Get the group names, groups_id, check if it's blacklisted.
 			$groupArr = [];
 			foreach ($file->groups->group as $group) {
-				$group = (string)$group;
+				$group = strtolower($group);
 
 				// If groups_id is -1 try to get a groups_id.
+				// why ask if $groupID === -1 when we did not change it since it was set it in line 281?
 				if ($groupID === -1) {
+                                        // test if group is a shortname
+                                        // $testgroup = "a.b.something";
+                                        if(preg_match("/^[a-zA-Z0-9]\.[a-zA-Z0-9]\./",$group)) {
+                                                echo "group $group is a shortname\n";
```

Comment:
Remove this, I assume you added it for testing?

Reply:
still testing yes